src/Components/Publishing/Article.tsx

```diff
@@ -33,6 +36,17 @@ export interface ArticleProps {
   onExpand?: () => void
 }
 
+export interface State {
+  showCtaBanner: boolean
+}
+
+export enum CtaCopy {
+  news = "Sign up for the best in art world news",
+  standard = "Sign up to get our best stories everyday",
+  feature = "Sign up to get our best stories everyday",
```

Comment:
Since news is the only one that differs from the default, we probably don't need to declare standard and feature here.